decidim-proposals/app/views/decidim/proposals/proposals/edit_draft.html.erb

```diff
@@ -42,8 +42,10 @@
             </div>
           <% end %>
 
-          <div class="actions">
-            <%= form.submit t(".send"), class: "button expanded", data: { disable: true } %>
+          <div class="row column flex-center">
+            <%= form.submit t(".send"), class: "button  button--nomargin small", data: { disable: true } %>
+
+            <%= link_to t(".discard"), destroy_draft_proposal_path(@proposal), method: :delete, data: { confirm: t(".discard_confirmation") } %>
```

Comment:
Should we make it look like a hollow button? Something like this:

![](https://i.imgur.com/mR9modO.png)

I'm not sure if we should, I'm just suggesting design here.

Reply:
In the issue #2838 is my proposal @carolromero agreed (with the discard button to the right), maybe @decidim/lot-px can help here.

Reply:
Oh, if product has already agreed then go for it 👍

Reply:
I agree @javierarce.
@carolromero or @decidim/product do you agree with the order: 
- left side: _discard proposal_ link
- right side: _publish_ button

Reply:
ping @decidim/product